flujo/validation/linters.py

```
@@ -1496,6 +1503,261 @@ def analyze(self, pipeline: Any) -> Iterable[ValidationFinding]:
+    as described in the FLUJO_TEAM_GUIDE.md "Fatal Anti-Pattern" section.
+
+    Control flow exceptions (PausedException, PipelineAbortSignal, InfiniteRedirectError)
+    must be re-raised to maintain proper workflow orchestration. Converting them to
+    StepResult(success=False) breaks pause/resume and other critical flows.
+    """
+
+    CONTROL_FLOW_EXCEPTIONS = {
+        "PausedException",
+        "PipelineAbortSignal",
+        "InfiniteRedirectError",
+    }
```

Ruff (0.13.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)